src/doc/src/reference/manifest.md

```diff
@@ -135,47 +135,84 @@ authors = ["Graydon Hoare", "Fnu Lnu <no-reply@rust-lang.org>"]
 This field is surfaced in package metadata and in the `CARGO_PKG_AUTHORS`
 environment variable within `build.rs` for backwards compatibility.
 
-### The `edition` field
+## The `[badges]` section
```

Comment:
Lexicographic order makes sense for a reference-like documentation. I believe we did it for `.cargo/config.toml` docs. However, I feel like fields in configuration are mostly equally-important but in Cargo.toml they are not. Moving an unused field upfront doesn't seem too good. 

Anyway, this is not a suggestion for you to apply. I am just sharing an example that I don't fully agree with the style guide.